src/System.Security.Permissions/src/System/Security/Permissions/ReflectionPermissionAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Permissions
+{
+    [System.AttributeUsageAttribute((System.AttributeTargets)(109), AllowMultiple = true, Inherited = false)]
+    public sealed partial class ReflectionPermissionAttribute : System.Security.Permissions.CodeAccessSecurityAttribute
+    {
+        public ReflectionPermissionAttribute(System.Security.Permissions.SecurityAction action) : base(default(System.Security.Permissions.SecurityAction)) { }
+        public System.Security.Permissions.ReflectionPermissionFlag Flags { get; set; }
+        public bool MemberAccess { get; set; }
+        [System.ObsoleteAttribute]
+        public bool ReflectionEmit { get; set; }
+        public bool RestrictedMemberAccess { get; set; }
+        [System.ObsoleteAttribute("not enforced in 2.0+")]
```

Comment:
Where did this string come from?  Ideally I think we'd just use the strings for the obsolete attributes that desktop uses, e.g. http://referencesource.microsoft.com/#mscorlib/system/security/permissions/permissionattributes.cs,475